src/Component/Customer/CustomerSelector.php

```diff
@@ -31,26 +32,37 @@ class CustomerSelector implements CustomerSelectorInterface
     protected $session;
 
     /**
-     * @var SecurityContextInterface
+     * @var AuthorizationCheckerInterface
      */
-    protected $securityContext;
+    protected $authorizationChecker;
+
+    /**
+     * @var TokenStorageInterface
+     */
+    protected $tokenStorage;
 
     /**
      * @var string
      */
     protected $locale;
 
     /**
-     * @param CustomerManagerInterface $customerManager
-     * @param SessionInterface         $session
-     * @param SecurityContextInterface $securityContext
-     * @param LocaleDetectorInterface  $localeDetector
+     * @param CustomerManagerInterface      $customerManager
+     * @param SessionInterface              $session
+     * @param AuthorizationCheckerInterface $authorizationChecker
+     * @param TokenStorageInterface         $tokenStorage
+     * @param LocaleDetectorInterface       $localeDetector
      */
-    public function __construct(CustomerManagerInterface $customerManager, SessionInterface $session, SecurityContextInterface $securityContext, LocaleDetectorInterface $localeDetector)
+    public function __construct(CustomerManagerInterface $customerManager,
```

Comment:
Please put the first and last argument on their own lines, this is not PSR2 compliant.

Reply:
Yep, just today read PSRs and installed cs-fixer not to be wrong with this anymore )